contractcourt/channel_arbitrator_test.go

```diff
@@ -801,29 +816,80 @@ func TestChannelArbitratorLocalForceClosePendingHtlc(t *testing.T) {
 		t.Fatalf("resolution msgs not sent")
 	}
 
+	// We'll grab the old notifier here as our resolvers are still holding
+	// a reference to this instance, and a new one will be created when we
+	// restart the channel arb below.
+	oldNotifier := chanArb.cfg.Notifier.(*mockNotifier)
```

Comment:
Is this still necessary now that the resolvers are reinstantiated?

Reply:
Unfortunately yes, since the arb log isn't re-created fully, so it passes a `ResolverKit` to the newly deserialized resolvers that still points to the arb config of the _old_ channel arb. The existing set also relied on this fact, as otherwise the channel arb would also be consuming the block epoch notifications, potentially robbing the resolvers of them.